[CI] 7.17 repository-azure:thirdPartyAudit failing

`:plugins:repository-azure:thirdPartyAudit` is [failing a lot on 7.17](https://gradle-enterprise.elastic.co/s/6rttmqvk64x6g/console-log?task=:plugins:repository-azure:thirdPartyAudit):

    > Task :plugins:repository-azure:thirdPartyAudit FAILED
    Unnecessary exclusions, following classes are not missing:
       * com.sun.org.apache.xml.internal.resolver.Catalog
      * com.sun.org.apache.xml.internal.resolver.tools.CatalogResolver

    FAILURE: Build failed with an exception.

    * What went wrong:
    Execution failed for task ':plugins:repository-azure:thirdPartyAudit'.
    > Third party audit task is not configured correctly

This reproduces locally with `./gradlew :plugins:repository-azure:thirdPartyAudit`

--- a/docs/changelog/92352.yaml
+++ b/docs/changelog/92352.yaml
@@ -1,0 +1,6 @@
+pr: 92352
+summary: Remove unnecessary `thirdPartyAudit` exclusions
+area: Infra/Core
+type: bug
+issues:
+ - 92346

--- a/plugins/repository-azure/build.gradle
+++ b/plugins/repository-azure/build.gradle
@@ -293,8 +293,6 @@ tasks.named("thirdPartyAudit").configure {
     )
   } else {
     ignoreMissingClasses(
-      'com.sun.org.apache.xml.internal.resolver.Catalog',
-      'com.sun.org.apache.xml.internal.resolver.tools.CatalogResolver',
       'javax.activation.DataHandler',
       'javax.activation.DataSource',
       'javax.xml.bind.JAXBElement',

